--- Symfony-orig/app/AppKernel.php	2012-03-19 01:03:15.000000000 +0100
+++ Symfony/app/AppKernel.php	2012-03-21 15:49:02.083019772 +0100
@@ -16,6 +16,11 @@
             new Symfony\Bundle\DoctrineBundle\DoctrineBundle(),
             new Symfony\Bundle\AsseticBundle\AsseticBundle(),
             new Sensio\Bundle\FrameworkExtraBundle\SensioFrameworkExtraBundle(),
+            new Jasny\FrameworkBundle\JasnyFrameworkBundle(),
+            new Jasny\ORMBundle\JasnyORMBundle(),
+            new Jasny\CKEditorBundle\JasnyCKEditorBundle(),
+            new Jasny\BootstrapBundle\JasnyBootstrapBundle(),
+            new Jasny\VanillaBundle\JasnyVanillaBundle(),
             new JMS\SecurityExtraBundle\JMSSecurityExtraBundle(),
         );
 
@@ -24,6 +29,7 @@
             $bundles[] = new Symfony\Bundle\WebProfilerBundle\WebProfilerBundle();
             $bundles[] = new Sensio\Bundle\DistributionBundle\SensioDistributionBundle();
             $bundles[] = new Sensio\Bundle\GeneratorBundle\SensioGeneratorBundle();
+            $bundles[] = new Jasny\GeneratorBundle\JasnyGeneratorBundle();
         }
 
         return $bundles;
@@ -32,5 +38,6 @@
     public function registerContainerConfiguration(LoaderInterface $loader)
     {
         $loader->load(__DIR__.'/config/config_'.$this->getEnvironment().'.yml');
+        $loader->load('@JasnyORMBundle/Resources/config/doctrine.yml');
     }
 }
